pkg/cmd/server/api/types.go

```diff
@@ -20,6 +20,10 @@ var (
 	DeadKubernetesAPILevels    = []string{"v1beta1", "v1beta2"}
 	DeadOpenShiftAPILevels     = []string{"v1beta1"}
 
+	FeatureAliases = map[string][]string{
```

Comment:
I don't have a use case for such a lookup because of additional helper maps/sets but it might come handy. I've just made those aliases lower-cased.